ci/parse_trivy_results_tests.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for get_duplicate_issue and process_vulnerabilities from
+parse_trivy_results.py.
+
+This will be run as a GitHub Action when parse_trivy_results.py has been
+modified.
+
+"""
+import unittest.mock as mock
+from absl import flags
+from absl.testing import absltest
+from parse_trivy_results import get_duplicate_issue, process_vulnerabilities
+
+FLAGS = flags.FLAGS
+
+
+class TestParseTrivyResults(absltest.TestCase):
+    def setUp(self):
+        FLAGS.release_tag = "0.0.0"
+        FLAGS.commit_id = -1
+
+        self.mock_issue_dup = mock.Mock()
+        self.mock_issue_dup.title = "Vulnerability found in release 0.0.0"
+        self.mock_issue_dup.body = "body"
+        self.mock_issue_dup.number = 0
+
+        self.trivy_json_empty = dict()
+        self.trivy_json_empty["Vulnerabilities"] = []
+
+        self.trivy_json_low = {"Vulnerabilities": [
+            {"Severity": "LOW"}], "Target": "Test Target"}
+
+        self.trivy_json_high = {"Vulnerabilities": [
+            {"Severity": "HIGH"}], "Target": "Test Target"}
+
+        self.trivy_json_mixed = {"Vulnerabilities": [
+            {"Severity": "LOW"}, {"Severity": "HIGH"}], "Target": "Test Target"}
+
+        self.mock_repo = mock.Mock()
+        self.mock_repo.create_issue.return_value = self.mock_issue_dup
+
+    def test_get_duplicate_issue_no_duplicates(self):
```

Comment:
Could you add a doc string describing what is being tested in each of these cases?